server/testutils/factories/referral.ts

```diff
@@ -40,10 +50,6 @@ export default ReferralFactory.define(() => ({
   offeringId: faker.string.uuid(),
   prisonNumber: faker.string.alphanumeric({ length: 7 }),
   referrerId: faker.string.numeric({ length: 6 }),
-  status: faker.helpers.arrayElement([
-    'awaiting_assesment',
-    'assessment_started',
-    'referral_started',
-    'referral_submitted',
-  ]) as ReferralStatus,
+  status,
+  submittedAt: status === 'referral_submitted' ? faker.date.past().toISOString() : undefined,
```

Comment:
I think this should be `status !== 'referral_started'`

Would this work if the status is passed in as a param? I think it might need to be `(params.status || status)` (there's similar in the course factory)